clients/src/main/resources/common/message/UpdateFeaturesRequest.json

```diff
@@ -29,8 +29,12 @@
         "about": "The name of the finalized feature to be updated."},
       {"name": "MaxVersionLevel", "type": "int16", "versions": "0+",
         "about": "The new maximum version level for the finalized feature. A value >= 1 is valid. A value < 1, is special, and can be used to request the deletion of the finalized feature."},
-      {"name": "AllowDowngrade", "type": "bool", "versions": "0+",
-        "about": "When set to true, the finalized feature version level is allowed to be downgraded/deleted. The downgrade request will fail if the new maximum version level is a value that's not lower than the existing maximum finalized version level."}
-    ]}
+      {"name": "AllowDowngrade", "type": "bool", "versions": "0",
```

Comment:
So if this is set to true and UpgradeType == 1 then we'll change UpgradeType to 2, right?

Reply:
I don't these can be set together. AllowDowngrade is just for version 0, and UpgradeType is version 1+

If you're talking about UpdateFeatureRequest, we just ignore UpgradeType when the message version is zero.